packages/web-components/src/components/footer/__stories__/footer.stories.ts

```diff
@@ -39,9 +48,11 @@ export const base = ({ parameters }) => {
             language="${ifNonNull(language)}"
             lang-display="${ifNonNull(langDisplay)}"
             size="${ifNonNull(size)}"
+            languageList="${ifNonNull(langList)}"
             .legalLinks="${ifNonNull(legalLinks)}"
             .links="${ifNonNull(links)}"
             .localeList="${ifNonNull(localeList)}"
+            .enableLanguageSelector="${enableLanguageSelector}"
```

Comment:
Please also see `@property` definition of `enable-language-selector`:

```suggestion
            ?enable-language-selector="${enableLanguageSelector}"
```